flocker/control/_persistence.py

```diff
@@ -0,0 +1,86 @@
+# Copyright Hybrid Logic Ltd.  See LICENSE file for details.
+
+"""
+Persistence of cluster configuration.
+"""
+
+from cPickle import dumps, loads
+
+from twisted.application.service import Service
+
+from ._model import Deployment
+
+
+# These should not use Pickle!@!
+# https://clusterhq.atlassian.net/browse/FLOC-1241x
+def serialize_deployment(deployment):
```

Comment:
Perhaps these are both private implementation details of `ConfigurationPersistenceService` (iow - underscore prefix?)